articles/operator-nexus/howto-configure-cluster.md

```diff
@@ -155,6 +155,70 @@ az networkcloud cluster deploy \
   --subscription "$SUBSCRIPTION_ID"
 ```
 
+> [!TIP]
+> To check the status of the `az networkcloud cluster deploy` command, it can be executed using the `--debug` flag.
+> This will allow you to obtain the `Azure-AsyncOperation` or `Location` header used to query the `operationStatuses` resource.
+> See the section [Cluster Deploy Failed](#cluster-deploy-failed) for more detailed steps.
+> Optionally, the command can run asynchronously using the `--no-wait` flag.
+
+### Cluster Deploy and Hardware Validation
```

Comment:
Suggest rephrasing to say "Deploy Cluster with optional hardware validation". 
Besides reading more consistently, also lowercase should be used on hardware validation.